src/js/_enqueues/wp/customize/preview.js

```diff
@@ -639,7 +639,7 @@
 		 * @return {void}
 		 */
 		custom_css: function( value ) {
-			$( '#wp-custom-css' ).text( value );
+			$( '#wp-custom-inline-css' ).text( value );
```

Comment:
This ID change could be problematic. There are plugins that are looking for `wp-custom-css` specifically: https://wpdirectory.net/search/01H9VJB3FFGA18GFZWQ95GZ1X6

Reply:
Seems like this is a blocker to making this change. This id comes from the style class. See https://github.com/WordPress/wordpress-develop/blob/2453a6b57d9bfeb0a5a5d191044b42d24955995b/src/wp-includes/class-wp-styles.php#L347

There is no filter or class property to change. I feel we have to revert this change :(

Reply:
Yeah, either that or heavy outreach to those plugins.